IFC4x3/Properties/n/NumberOfTractionPoints_1Nk$_0PhT2kBz$hK6Rfjvq/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfTractionPoints_1Nk_0PhT2kBzhK6Rfjvq" Name="NumberOfTractionPoints" UniqueId="57bbff80-66b7-42b8-bf7f-ad419ba6de74" PrimaryDataType="IfcCountMeasure" />
```

Comment:
Seeing that this property is on actuator, is this not already possible to derive from the model?